src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuite.Table.al

```diff
@@ -51,11 +51,18 @@ table 149030 "AIT Test Suite"
             var
                 AITTestMethodLine: Record "AIT Test Method Line";
             begin
+                if Rec."Input Dataset" = xRec."Input Dataset" then
```

Comment:
This is not good if modified from code it will always be the same value. Double check this because it is validate trigger but I believe if started from AL code then it will have the same value. 

We should not use xRec outside of the page context. To fix if they are the same do the xRec.FiND and then if they are the same exit. Othewise run the code. Alternative is to delete the check.

Reply:
Moved the OnValidate trigger to the page. Probably also make more sense only to trigger when modified on the page.